thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datalayer/pojo/ClassificationConfigBean.java

```diff
@@ -0,0 +1,42 @@
+package com.linkedin.thirdeye.datalayer.pojo;
+
+import java.util.List;
+
+public class ClassificationConfigBean extends AbstractBean {
```

Comment:
Can we have some java doc explaining this new config and these fields? mainFunctionId isn't too self explanatory.